libraries/helpers.rb

```diff
@@ -16,12 +16,11 @@ def raw_checksums_to_hash
     # Downloads the raw checksums from hashicorp
     # https://releases.hashicorp.com/terraform/#{version}/#{checksum_file}
     def fetch_checksums
-      require 'uri'
       uri = URI.parse(node['terraform']['url_base'])
-      terraform_releases = "#{uri.scheme}://#{uri.host}/"
+      terraform_releases = "#{uri.scheme}://#{uri.host}"
       version = node['terraform']['version']
       checksum_file = "terraform_#{version}_SHA256SUMS"
-      path = "#{uri.path}/#{version}/#{checksum_file}"
+      path = "terraform/#{version}/#{checksum_file}"
```

Comment:
The path can't have `terraform` hardcoded here, because someone might need to wrap this cookbook or override the value for `node['terraform']['url_base']`. We can't assume that their URL will start with `terraform` in the path; hence, the previous version used `uri.path` instead.